templates/mission-conops.adoc.liquid

```diff
@@ -48,10 +48,10 @@ As a *{{us.actor}}* I want to *{{us.behavior}}* so that I can *{{us.rationale}}*
 
 {{us.example}}
 
-{% if us.derivedFrom.length > 0 %}
+{% if ds.derivedFrom.length > 0 %}
 ==== *Derived From:*
 
-{% for derivedFromItem in us.derivedFrom %}
+{% for derivedFromItem in ds.derivedFrom %}
```

Comment:
this should remain as `us.derivedFrom`, since it's pulling the derivedFrom elements from the user stories, not data structures